### PR TITLE
Create test_iso639.py

### PR DESCRIPTION
**Pull request by william1835**
Written a test module for the iso639 module. 
Includes changing http to https URLs in iso639 and ethnologue modules, because they were causing a bit of nuisance with ResourceWarning, and were being redirected.
Tell me if I've missed anything, or it's not written how you would like it to be written.

### CODE REVIEW

File: modules/test/test_iso639.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        for mock in mocks:
+            mock.reset_mock()
+       
+    def iso3_to_iso1(self, code):
+        for iso1, iso3 in self.iso1_to_iso3.items():
+            if code == iso3:
+                return iso1
+        return ''    
+    
+    def set_phenny_say_return_value(self, arg):
```
Comment (wei2912) [inline modules/test/test_iso639.py]: avoid using setters in python

File: modules/test/test_iso639.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        self.phenny_say_return_value = arg
+       
+    def test_iso_code_scrape(self):
+        for k, v in self.iso_to_lang.items():
+            self.assertEqual(self.phenny.iso_data[k], v)
+            
+    def test_iso_code_convert_scrape(self):
+
```
Comment (wei2912) [inline modules/test/test_iso639.py]: blank line

File: modules/test/test_iso639.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+            self.phenny.say.assert_called_once_with(
+                iso3 + ', ' + self.iso3_to_iso1(iso3)
+                + ' = ' + self.iso_to_lang[iso3]
+            )
+            self.input.group.assert_called_once_with(2)
+            self.reset_mock(self.phenny, self.input)
+    
+    def test_iso_to_lang_write_dict(self):
+        try:
+            file = open(self.iso_filename,'r')
```
Comment (wei2912) [inline modules/test/test_iso639.py]: use `f` instead of `file`

File: modules/test/test_iso639.py [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    def test_iso_to_lang_read_dict(self):
+        try:
+            iso_data = iso639.read_dict(self.iso_filename)
+            self.assertEqual(iso_data, self.phenny.iso_data, 'ISO data written is not the same as the scraped ISO data')
+        except IOError:
+            self.fail("ISO database was not written at all")
+            
+    def test_iso_one_to_three_write_dict(self):
+        try:
+            file = open(self.iso_one_to_three_filename,'r')
```
Comment (wei2912) [inline modules/test/test_iso639.py]: use `f` instead of `file`

File: modules/test/test_iso639.py [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+            iso639.iso639(self.phenny, self.input)
+            for k, v in self.phenny.iso_data.items():
+                if lang in v:
+                    self.assertIn(k + ' = ' + v, self.phenny_say_return_value)
+        
+    @classmethod
+    def tearDownClass(cls):
+        os.remove(os.path.expanduser(cls.iso_filename))
+        os.remove(os.path.expanduser(cls.iso_one_to_three_filename))
+
```
Comment (wei2912) [inline modules/test/test_iso639.py]: delete trailing spaces but make sure there's a newline at the end of the file

File: modules/test/test_iso639.py (hunk 1 above)
Comment (william1835) [inline modules/test/test_iso639.py]: How would I correct this?

**Pull request merged**